Dockerfile

```diff
@@ -9,6 +9,8 @@ RUN pip install deepdiff==1.7.0
 RUN pip install pytest-cov hsmodels
 RUN pip install --upgrade rdflib==5.0.0
 
+RUN pip install sorl-thumbnail
```

Comment:
[sorl-thumbnail](https://github.com/jazzband/sorl-thumbnail) uses Redis cache to quickly return thumbnails of images.

Reply:
@devincowan Will this cache setup automatically remove entries of files that have been deleted? For example, if I change a group picture will cached images corresponding to that file be deleted eventually?

Reply:
@Maurier yes, the Sorl-thumbnail manages the cache and removes entries when files get deleted.